components/builder-db/src/migration.rs

```diff
@@ -17,22 +17,15 @@ use diesel::query_dsl::RunQueryDsl;
 use diesel::sql_query;
 use error::Result;
 
-pub fn shard_setup(conn: &PgConnection, shard_id: u32) -> Result<()> {
-    debug!("Migrating shard_{:?}", shard_id);
-    sql_query(format!("CREATE SCHEMA IF NOT EXISTS shard_{}", shard_id))
-        .execute(conn)
-        .unwrap();
-    sql_query(format!("SET SEARCH_PATH TO shard_{}", shard_id))
-        .execute(conn)
-        .unwrap();
+pub fn shard_setup(conn: &PgConnection) -> Result<()> {
```

Comment:
Maybe rename this to just setup or setup_ids ?

Reply:
Will do